### PR TITLE
Holistic style refactor and content update

### PR DESCRIPTION
**Pull request by CarsonBurke**
## Primary changes

-favicon
-parallax
-change font to poppins
-increase some font sizes
-stylize overview
-distinct overview images
-banner

## Media

image
image
image
image
image

### CODE REVIEW

File: images/chocolatey/icon.png
Comment (Mailaender) [inline images/chocolatey/icon.png:1]: The icon is used for https://community.chocolatey.org/packages/openhv which requires these exact dimensions, so don't change it, but duplicate it for the web.

File: index.html [hunk 1]
```diff
@@ -8,88 +8,107 @@
...
+            <div class="solidToImage"></div>
+            <section class="overviewParent overviewLeft gapMedium">
+
+                <div class="showcaseChild overviewContent gapMedium">
                     <div class="gapSmall centerColumn column">
                         <h3 class="h2">Classic RTS</h3>
                         <p class="p2 showcaseDescription">
-                            Journey back to the golden age of real-time strategy in a modern engine with todays convience and features.
+                            Journey back to the golden age of real-time strategy in a modern engine with todays
+                            convience and features.
```
Comment (Mailaender) [inline index.html:67]: Why do you do these line breaks in the middle of the sentence?

File: images/web/combat.mp4
Comment (Mailaender) [inline images/web/combat.mp4:1]: Can you squash this?

File: index.html (hunk 1 above)
Comment (CarsonBurke) [reply index.html:67]: My formatter does this to limit the line length so that you don't have to scroll a ton

File: images/chocolatey/icon.png
Comment (CarsonBurke) [reply images/chocolatey/icon.png:1]: understood, will do

File: images/web/combat.mp4
Comment (CarsonBurke) [reply images/web/combat.mp4:1]: I think that's at the extent of my git knowledge, sorry

**Pull request closed without merging**